code/modules/station_goals/meteor_shield.dm

```diff
@@ -107,7 +107,7 @@
 			continue
 		if(space_los(meteor_to_destroy))
 			var/turf/beam_from = get_turf(src)
-			beam_from.Beam(get_turf(meteor_to_destroy), icon_state="sat_beam", time = 5)
+			beam_from.Beam(get_turf(meteor_to_destroy), icon_state="sat_beam", time = 3)
```

Comment:
This time value is only a visual effect on the beam that determines how long the beam stays for, however it doesn't affect how fast the meteor shield shoots.

Also it's in deciseconds, not seconds.